runtime/shared_common/shrinit.cpp

```diff
@@ -4772,6 +4776,10 @@ j9shr_jvmPhaseChange(J9VMThread *currentThread, UDATA phase)
 	if (J9VM_PHASE_NOT_STARTUP == phase) {
 		J9JavaVM* vm = currentThread->javaVM;
 
+		/* OpenJ9 issue; https://github.com/eclipse/openj9/issues/3743
+		 * New GC API j9gc_storeHeapSizes decides whether to calls vm->sharedClassConfig->storeGCHints() to store the GC hints into the shared cache. */
```

Comment:
I don't think this should refer to an API in another component that may or may not exist later.

Reply:
Fixed